eduiddashboard/__init__.py

```diff
@@ -394,7 +394,7 @@ def main(global_config, **settings):
     )
 
     try:
-        settings['session.cookie_max_age'] = int(settings.get('session.cookie_max_age', 3600))
+        settings['session.cookie_max_age'] = int(settings.get('session.cookie_max_age', 600))
```

Comment:
Yes, 10 min seems to me a more sensible setting for the session ttl than an hour, but I'm probably wrong. The session ttl is reset in each request.